data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```
@@ -75,5 +75,55 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+    "id": "cleansingflame_hermes_heal",
+    "type": "SPELL",
+    "spell_class": "CLEANSING_FLAME",
+    "name": "Hermes Reborn",
+    "description": "In the sea without lees\nStandeth the bird of Hermes\nEating his wings variable\nAnd maketh himself yet full stable",
+    "valid_targets": [ "self" ],
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 350,
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "difficulty": 10,
+    "min_damage": -20,
+    "max_damage": -120,
+    "damage_increment": -2.5,
+    "max_level": 40,
+    "extra_effects": [ { "id": "cleansingflame_hermes_burn" }, { "id": "cleansingflame_hermes_split" } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "NO_PROJECTILE", "IGNORE_WALLS", "LOUD", "MUST_HAVE_CLASS_TO_LEARN" ]
+  },
+  {
+    "id": "cleansingflame_hermes_burn",
+    "type": "SPELL",
+    "name": "Hermes Reborn fire",
+    "description": "Give some frost damage for Hermes Reborn spell.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "self", "ally", "hostile", "ground" ],
+    "flags": [ "LOUD", "NO_PROJECTILE" ],
+    "effect": "attack",
+    "shape": "blast",
+    "min_damage": 10,
+    "max_damage": 25,
+    "damage_increment": 0.4,
+    "min_range": 1,
+    "max_range": 1,
+    "damage_type": "cold",
+    "min_aoe": 3,
+    "max_aoe": 5,
+    "aoe_increment": 0.05,
+    "spell_class": "CLEANSING_FLAME",
```

```suggestion
    "spell_class": "CLEANSING_FLAME"
```
Alternatively, remove entirely and also delete the comma from the prior line.